0009-non-helium-hotspots.md

```diff
@@ -0,0 +1,236 @@
+encompasses all the benefits and constraints of the lower Levels (with a few exceptions).
+
+                                                        +------------+
+                                                        |            |
+                                                +-------+  Level 3A  +---------+
+                                                |       |            |         |
+    +-----------+         +-----------+         |       +------------+         |         +-----------+
+    |           |         |           |         |                              |         |           |
+    |  Level 1  +-------->+  Level 2  +-------->+                              +-------->+  Level 4  |
+    |           |         |           |         |                              |         |           |
+    +-----------+         +-----------+         |       +------------+         |         +-----------+
+                                                |       |            |         |
+                                                +-------+  Level 3B  +---------+
+                                                        |            |
+                                                        +------------+
+
+### Level I: Getting on the network
+
+- Every hotspot which joins the helium network starts at Level 1 by issuing an `add_gateway` transaction.
+- The entry fee is set to burning data credits worth $10 USD.
+- A hotspot at Level 1 does not have any location asserted on the network, however, it is allowed to witness nearby PoC
+  challenges. This is to ensure that when the hotspot does assert a location via `assert_location` transaction, we can
+  be confident about its claim of location by analyzing its witness list.
+- A hotspot at Level 1 is allowed to transmit data and earn HNT based only on transmitting network data.
+
+### Level 2: Asserting location on the network
+
+- A hotspot must satisfy the below minimum criteria to enter Level 2:
+    * It must be a Level 1 hotspot for X number of blocks.
+    * Issue an `assert_location` transaction by incurring a $40 USD data credit burn fee.
+    * Witness sufficient number of PoC challenges involving Level 3 hotspots.
+- A hotspot at Level 2 gains access to participate in PoC challenges (become a member in the PoC path).
+
+### Level 3: Gaining PoC challenge priviledges
+
+Level 3 is divided into two broad categories:
+
+#### Level 3A: Organic network growth
+
+- A hotspot must satisfy the below minimum criteria to enter Level 3A:
+    * It must be a Level 2 hotspot for X number of blocks.
+    * Successfully participate in X number of PoC challenges involving other Level 3 hotspots.
+    * A successful challenge comprises of the hotspot being able to receive a PoC packet from a Level 3 hotspot,
+      successfully decrypt it and successfully transmit it to a Level 2 or above hotspot.
+- A hotspot at Level 3A gains access to construct PoC challenges.
+
+#### Level 3B: On-chain governance
```

Comment:
it would be nice, if I am reading this correctly, to make it clearer that this is an alternate route to being a L3 spot?

Reply:
yeah, certainly could be worded better